src/select.coffee

```diff
@@ -400,6 +400,9 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
     )
   .property 'groupedContent.[]'
 
+  # whether or not to ensure that highlighted option is visible
```

Comment:
how about `If yes, the highlighted selection option is always visible. This may result in a 'jump': when there are duplicated options and mouse hovers over the second occurrence of the option, it will jump to the first occurrence.`